--- gmpc-lyrics-0.14.0/src/Makefile.am.wig	2007-01-11 23:50:36.000000000 +0100
+++ gmpc-lyrics-0.14.0/src/Makefile.am	2007-02-20 19:54:39.000000000 +0100
@@ -1,7 +1,6 @@
-#bin_PROGRAMS=wikiplugin_la
-libdir=${prefix}/share/gmpc/plugins/
+pluginsdir=$(libdir)/gmpc/
 
-lib_LTLIBRARIES = lyricsplugin.la
+plugins_LTLIBRARIES = lyricsplugin.la
 
 lyricsplugin_la_SOURCES= plugin.c _leoslyrics.c _lyrictracker.c _lyricwiki.c
 
@@ -11,9 +10,3 @@
 lyricsplugin_la_LDFLAGS= -module -avoid-version
 
 EXTRA_DIST= gmpc-lyrics.h
-
-install:
-	@$(MAKE)
-	mkdir -p ${DESTDIR}${libdir}
-	if test -f .libs/lyricsplugin.so; then cp .libs/lyricsplugin.so ${DESTDIR}${libdir}; fi
-	if test -f .libs/lyricsplugin.dll; then cp .libs/lyricsplugin.dll ${DESTDIR}${libdir}; fi
